### PR TITLE
Output collection

### PR DESCRIPTION
**Pull request by romainfd**
Final results:
image

Excel:
[output (2).xlsx](https://github.com/Vin0uz/cutiro-api/files/5899377/output.2.xlsx)

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -43,3 +62,31 @@ def default(self, obj):
...
+  # Remove duplicates from payroll
+  if 'payroll_duplicates' in event:
+    # Drop matched duplicates
+    for _, removed in event['payroll_duplicates']:
+      payroll = payroll.drop(removed)
+  # Convert matches to Series and left join them
+  duplicates = pd.Series(event['payroll_duplicates'], name="Duplicats de Payroll")
+  payroll = payroll.join(duplicates)
+  # Matched teachers
+  cols = ['', 'Matchs certains', 'Matchs confiants', 'Matchs possibles', 'Ghost teachers']
```
Comment (Vin0uz) [inline utils.py:79]: Why the first col is `''`?

Comment (romainfd) [reply utils.py:79]: Just because we start counting with `step1` and not `step0`

Comment (romainfd) [reply utils.py:79]: It's just like doing `f"step{i-1}"` below

Comment (Vin0uz) [reply utils.py:79]: That's what I suspected, ok 👌

**Review by Vin0uz — APPROVED** [review]

**Pull request merged**